src/pipeline/query_pipeline.rs

```diff
@@ -180,13 +180,27 @@ impl RawQueryPipeline {
         colliders: &RawColliderSet,
         point: &RawVector,
         groups: u32,
+        filter: &js_sys::Function,
     ) -> Option<RawPointColliderProjection> {
+        let filtercb = |handle: ColliderHandle| match filter.call1(
+            &JsValue::null(),
+            &JsValue::from(handle.into_raw_parts().0 as u32),
+        ) {
+            Err(_) => true,
+            Ok(val) => val.as_bool().unwrap_or(true),
+        };
+        let rfilter: Option<&dyn Fn(ColliderHandle) -> bool> = if filter.is_function() {
+            Some(&filtercb)
+        } else {
+            None
+        };
```

Comment:
We should use `wrap_filter` like in the other queries to simplify this.

Reply:
ahhh~,   got it, sorry, didn't notice that at binding side.